flink-state-backends/flink-statebackend-rocksdb/pom.xml

```diff
@@ -36,6 +36,24 @@ under the License.
 
 	<packaging>jar</packaging>
 
+	<repositories>
+		<repository>
+			<id>sonatype</id>
+			<name>Sonatype repository</name>
```

Comment:
Does this repository must exist?

Reply:
This is only so that we can verify that the published package works as we expect to before publishing the Maven. I'll remove it before the PR is ready to merge.